### ISSUE TITLE
$evaluate-measure uses patient input parameter but should be subject

### ISSUE THREAD

**Issue by p9g** [DBCG/cqf-ruler#430]
https://github.com/DBCG/cqf-ruler/blob/master/r4/src/main/java/org/opencds/cqf/r4/providers/MeasureOperationsProvider.java#L186 
@OperationParam(name = "patient") String patientRef  
STU3 used "patient", but R4 changed to "subject"
subject should allow both references to either Patient or Group resources

**Comment by p9g** [comment]
https://github.com/DBCG/cqf-ruler/issues/170 is related but different

**Comment by p9g** [comment]
Note that $care-gaps does use "subject" https://github.com/DBCG/cqf-ruler/blob/master/r4/src/main/java/org/opencds/cqf/r4/providers/MeasureOperationsProvider.java#L267

And it supports Patient or Group
https://github.com/DBCG/cqf-ruler/blob/master/r4/src/main/java/org/opencds/cqf/r4/providers/MeasureOperationsProvider.java#L267

**Comment by rob-reynolds** [comment]
Hello @p9g.  This is being fixed on a large refactor branch that will be merged (relatively) soon: https://github.com/DBCG/cqf-ruler/blob/82301f0c568d3b472ea9713d86e1545bc800efb7/plugin/cr/src/main/java/org/opencds/cqf/ruler/cr/r4/provider/MeasureEvaluateProvider.java#L72

Please look for that.

**Comment by rob-reynolds** [comment]
I agree with the Patient or Group thing, like $care-gaps, but we had not planned on that.  

Looking at the R4 spec, it doesn't explicitly state it, but it doesn't preclude it either: https://www.hl7.org/fhir/operation-measure-evaluate-measure.html.

@JPercival @brynrhodes What do you think about adding group to $evaluate-measure?  I like the idea.

**Comment by JPercival** [comment]
Already done on the refactor branch, defined in the ATR IG.

**Comment by rob-reynolds** [comment]
Well that solves it then!
